microsite/data/plugins/home.yaml

```
@@ -0,0 +1,11 @@
+---
+title: Home
+author: '@Spotify'
```

```suggestion
author: Spotify
```
All of our plugins shown (except GRM) just have `Spotify` here.

---

> We should probably get our graphics department to make a nice logo here 😅 I can reach out to them.

Haha, uploaded a new one now!